docs/development.rst

```diff
@@ -94,10 +95,13 @@ Run the Tests
 #. Install test requirements::
 
     pip install -r requirements-test.txt
+    npm install -g gulp
```

Comment:
Gulp is installed locally using `npm install` on the following line. You shoudn't need it globally as well.

Since we're currently only using Gulp for running JSHint, we could probably even run JSHint using the CLI directly using something like this:

```sh
$ ./node_modules/.bin/jshint *.js dashboard/static/js/*.js

dashboard/static/js/analytics.js: line 78, col 63, Expected '===' and instead saw '=='.
dashboard/static/js/analytics.js: line 86, col 49, Expected '===' and instead saw '=='.

2 errors
```

(Ignore the errors, I was testing globs below to make sure everything was kosher.)

We can further optimize by adding something like this in the package.json file:

```json
"scripts": {
  "test": "jshint *.js dashboard/static/js/*.js"
}
```

Then just running `$ npm test`, and then we won't need to know about that weird "./node_modules/.bin/" prefix since that is automatically added by npm when using scripts.